lib/chef/knife/cookbook_site_share.rb

```diff
@@ -18,22 +18,19 @@
 #
 
 require "chef/knife"
-require "chef/mixin/shell_out"
 
 class Chef
   class Knife
     class CookbookSiteShare < Knife
 
-      include Chef::Mixin::ShellOut
-
       deps do
         require "chef/cookbook_loader"
         require "chef/cookbook_uploader"
         require "chef/cookbook_site_streaming_uploader"
-        require "mixlib/shellout"
-      end
+        require "chef/mixin/shell_out"
 
-      include Chef::Mixin::ShellOut
+        include Chef::Mixin::ShellOut
```

Comment:
does that still work the same way?  `include` needs `self` to be the class, and if `deps` gets called by the instance then this would behave...  strangely...